staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -120,16 +121,18 @@ type Interface interface {
 type MutationInterface interface {
 	Interface
 
-	// Admit makes an admission decision based on the request attributes
-	Admit(a Attributes, o ObjectInterfaces) (err error)
+	// Admit makes an admission decision based on the request attributes.
+	// Context is used only for timeout/deadline and tracing information.
```

Comment:
Looks like cancelations are supported by the underlying rest client as well?

Reply:
Will update the comment